src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -264,6 +264,14 @@ export class ESLToggleable extends ESLBaseElement {
     targetEl.setAttribute('aria-hidden', String(!this._open));
   }
 
+  /** @returns if the passed event should trigger hide action */
+  public isOutsideAction(e: Event): boolean {
+    const target = e.target as HTMLElement;
+    if (this.contains(target)) return false;
+    if (this.activator && this.activator.contains(target)) return false;
```

Comment:
cosmetic : I think it reads well too
```suggestion
    if (this.activator && this.activator.contains(target) || this.contains(target)) return false;
```

Reply:
Doesn't change the style because of readability reasons. Add comments instead